examples/getting_started/renate_config.py

```diff
@@ -1,9 +1,10 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
-from typing import Callable, Literal, Optional
+from typing import Callable, Literal, Optional, Dict
```

Comment:
nit: sort. you can enable that in pycharm so you would no longer need to care of it